### PR TITLE
Thunderbolt docking station documentation

### PR DESCRIPTION
**Pull request by Pokisiekk**

### CODE REVIEW

File: docs/unified-test-documentation/dasharo-compatibility/340-docking-station-net-interface.md [hunk 1]
```diff
@@ -36,8 +36,6 @@ Ethernet port can be obtained on Ubuntu 22.04.
 1. Power on the DUT.
 1. Boot into the system.
 1. Log into the system by using the proper login and password.
-1. Connect docking station to power source.
```
Comment (sulewskiprzemyslaw) [inline docs/unified-test-documentation/dasharo-compatibility/340-docking-station-net-interface.md:39]: @Pokisiekk If we don't want to write about connecting the docking station during the test, please add information about needed connections in the section `Test setup`.

File: docs/unified-test-documentation/dasharo-compatibility/340-docking-station-net-interface.md [hunk 2]
```diff
@@ -88,8 +86,6 @@ Ethernet port can be obtained on Windows 11.
 1. Power on the DUT.
 1. Boot into the system.
 1. Log into the system by using the proper login and password.
```
Comment (sulewskiprzemyslaw) [inline docs/unified-test-documentation/dasharo-compatibility/340-docking-station-net-interface.md:88]: As above. Please review all documentation and determine if such information would need to be added.

**Review by sulewskiprzemyslaw — CHANGES_REQUESTED** [review]

File: docs/unified-test-documentation/dasharo-compatibility/340-docking-station-net-interface.md (hunk 1 above)
Comment (Pokisiekk) [reply docs/unified-test-documentation/dasharo-compatibility/340-docking-station-net-interface.md:39]: Information is in the test setup

File: docs/unified-test-documentation/dasharo-compatibility/340-docking-station-net-interface.md (hunk 2 above)
Comment (Pokisiekk) [reply docs/unified-test-documentation/dasharo-compatibility/340-docking-station-net-interface.md:88]: I have reviewed and added information where needed

**Review by sulewskiprzemyslaw — APPROVED** [review]

**Pull request merged**